ui/src/app/cohort-search/redux/actions/creators.ts

```diff
@@ -291,6 +300,21 @@ export const clearWizardFocus =
   (): ActionTypes[typeof CLEAR_WIZARD_FOCUS] =>
   ({type: CLEAR_WIZARD_FOCUS});
 
+export const hideGroup =
+  (groupId: string, status: string
+  ): ActionTypes[typeof HIDE_GROUP] =>
+    ({type: HIDE_GROUP, groupId, status});
```

Comment:
when hiding a search group, it seems to show a count of 0 or empty when another group is present.

Reply:
Changed all to empty, may change again based on feedback

Reply:
Changed total count back to 0 based on CB team recommendation